libs/pmp-web/repository/shell/src/lib/shell-routing.module.ts

```diff
@@ -13,6 +13,13 @@ const routes: Routes = [
           import('@pimp-my-pr/pmp-web/repository/repositories-statistics/feature').then(
             m => m.PmpWebRepositoryRepositoriesStatisticsFeatureModule
           )
+      },
+      {
+        path: ':repositoryName',
```

Comment:
I think it hast to be :owner/:repositoryName

Reply:
I don't want change it until I'll know exactly data to get from routing

Reply:
But this is on your side to define what data do you need and I believe that just a repository name is not enough.

Reply:
I think it is enough because here is pass the repository number what as I assume is unique. So I will change the parameter's name to suitable